challenges/largest_product_array/test_largest_product.py

```diff
@@ -0,0 +1,18 @@
+import largest_product.py
```

Comment:
This should just be `largest_product` without the file ending or your import will not work properly.